arbeitszeit_web/www/presenters/end_cooperation_presenter.py

```diff
@@ -17,13 +17,14 @@ class ViewModel:
         show_404: bool
         redirect_url: str
 
-    request: Request
     notifier: Notifier
     url_index: UrlIndex
     translator: Translator
     session: Session
 
-    def present(self, response: EndCooperationResponse) -> ViewModel:
+    def present(
+        self, response: EndCooperationResponse, *, web_request: Request
```

Comment:
shouldn't we use named parameters everywhere by convention? why do you like to start forcing it here?